okapi-common/src/main/java/org/folio/okapi/common/XOkapiHeaders.java

```diff
@@ -197,4 +197,10 @@ private XOkapiHeaders() {
    * version.
    */
   public static final String OKAPI_MODULE = "okapi";
+
+  /**
+   * Cookie acess token name.
+   */
+  public static final String COOKIE_ACCESS_TOKEN = "accessToken";
```

Comment:
wiki.example.com might set an accessToken for *.example.com that will be set for okapi.example.com.

To avoid a name clash the cookie name should be more specific, for example folioAccessToken.

Reply:
I made this change and pushed it here so that we could have all the branches aligned for testing. See what you think: https://github.com/folio-org/okapi/pull/1251/commits/ed437e12699d3bad1c6ba45ed2342e5f6f85f890.

Going to now make this change in the other PRs that are affected.